src/main/kotlin/reposfinder/config/WorkConfig.kt

```diff
@@ -0,0 +1,21 @@
+package reposfinder.config
+
+data class WorkConfig(
+    val dumpDir: String,
+    val token: String? = null, // !!!
```

Comment:
What `!!!` mean? If it something that we shouldn't forget in the future it's better to comment on it.
Also, we can oblige users to pass tokens, so this field wouldn't be null.